### ISSUE TITLE
Has it already been updated to version 10.1 for use in the Docker environment?

### ISSUE THREAD

**Issue by mario-evangelista-silva** [WWBN/AVideo#4381]
Has it already been updated to version 10.1 for use in the Docker environment?
What are the procedures? Something special?

**Comment by DanielnetoDotCom** [comment]
Sorry I am not supporting docker version

**Comment by mario-evangelista-silva** [comment]
So will it stop at version 10 on the link https://github.com/WWBN/AVideo?

**Comment by DanielnetoDotCom** [comment]
I am not sure but I guess you can make a git pull inside the docker exec.

You may want to ask for some help from the ones who add this feature 

https://github.com/WWBN/AVideo/commits/master/Dockerfile

Maybe @gnovaro or @M-Ahadi

**Comment by mario-evangelista-silva** [comment]
OK thank you. I'll try.

**Comment by mario-evangelista-silva** [comment]
I was able to update successfully.
I made a new **git clone** of the site at the link https://github.com/WWBN/AVideo. Then I ran it again: **docker-compose up --build -d**.
When I reactivated the avideo, I accessed the update module and a pop-up message appeared informing me if I want to update to version 10.1, I confirmed and it was updated. Then, when accessing the **Plugins Module**, make the updates requested in the modules you request.

**Note: I am using persistent docker volumes, which map the "/ var / www / avideo / videos" and "/ var / lib / mysql" directories to a local directory on Host Docker. This mapping is done in your modified "docker-compose.yml" file that previously had these mappings for the host's local directory.**

Successfully updated to version 10.1 in the Docker environment.
